mlflow/pyfunc/__init__.py

```diff
@@ -868,14 +872,16 @@ def spark_udf(spark, model_uri, result_type="double", env_manager="local"):
     should_use_spark_to_broadcast_file = not (is_spark_in_local_mode or should_use_nfs)
     env_root_dir = _get_or_create_env_root_dir(should_use_nfs)
 
+    result_type = "boolean" if result_type == "bool" else result_type
```

Comment:
"boolean" is not a python type. Added a resolver here for usability / prevent users from getting annoyed.

Reply:
This is also intended to mirror how feature store is handling `score_batch()` with resolving `bool` -> `boolean`.